client/public/clientJS/heroTyping.js

```diff
@@ -0,0 +1,62 @@
+class TypeWriter {
+    constructor(txtElement, words, wait = 2000) {
+        this.txtElement = txtElement;
+        this.words = words;
+        this.txt = '';
+        this.wordIndex = 0;
+        this.wait = parseInt(wait, 10);
+        this.type();
+        this.isDeleting = false;
+    }
+
+    type() {
+        // Current index of word
+        const current = this.wordIndex % this.words.length;
+        // Get full text of current word
+        const fullTxt = this.words[current];
+        // Check if deleting
+        if (this.isDeleting) {
+            // Remove char
+            this.txt = fullTxt.substring(0, this.txt.length - 1);
+        } else {
+            // Add char
+            this.txt = fullTxt.substring(0, this.txt.length + 1);
+        }
+        // Insert txt into element
+        this.txtElement.innerHTML = `<span>${this.txt}</span>`;
+
+        // Initial Type Speed
+        let typeSpeed = 180;
+
+        if (this.isDeleting) {
+            typeSpeed /= 2;
+        }
+        // If word is complete
+        if (!this.isDeleting && this.txt === fullTxt) {
+            // Make pause at end
+            typeSpeed = this.wait;
+            // Set delete to true
+            this.isDeleting = true;
+        } else if (this.isDeleting && this.txt === '') {
+            this.isDeleting = false;
+            // Move to next word
+            this.wordIndex++;
+            // Pause before start typing
+            typeSpeed = 350;
+        }
+        setTimeout(() => this.type(), typeSpeed);
+    }
+}
```

Comment:
this can be achieved without the need of a class, I'd personally just put everything into the type function